app/src/main/java/bankingapp/App.java

```diff
@@ -39,18 +40,65 @@ public static void main(String[] args) {
                             "0. Exit");
 
                     try {
-                        response = input.nextInt();
+                        response = Integer.parseInt(input.next());
                         switch (response) {
+
                             case 1:
-                                System.out.println(new CreateAccount(connection));
+                                //Create a new user account
+                                String firstName;
+                                String lastName;
+
+                                 //Prompt user input for first and last names
+                                 System.out.println("Enter First Name: ");
+                                 do {
+                                    firstName = input.nextLine();
+                                 } while (firstName.equals(null) || firstName.trim().isEmpty());
+
+                                 System.out.println("Enter Last Name: ");
+                                 do {
+                                    lastName = input.nextLine();
+                                 }while (lastName.equals(null) || lastName.trim().isEmpty());
+
+                                //Create a new account object
+                                CreateAccount account = new CreateAccount();
+
+                                //Create a new account number and pin
+                                account.setAccountNumber();
+                                account.setPinNumber();
+
+                                //Display new user details
+                                System.out.println(new CreateAccount(firstName, lastName, account.getAccountNumber(),
+                                        account.getPinNumber()));
+
+                                //Input new user info into DB
+                                DBTrans.insertDB(connection,firstName,lastName, account.getAccountNumber(),
+                                        String.valueOf(account.getPinNumber()));
                                 break;
+
                             case 2:
-                                new LogIn(connection);
+                                //login to an existing account
+                                String accountNumber;
+                                String pinNumber;
+
+                                //Prompt user input for first and last names
+                                System.out.println("\nEnter your account number:");
+                                do {
+                                    accountNumber = input.nextLine();
+                                } while (accountNumber.equals(null) || accountNumber.trim().isEmpty());
+
+                                System.out.println("Enter your PIN:");
+                                do {
+                                    pinNumber = input.nextLine();
+                                } while (pinNumber.equals(null) || pinNumber.trim().isEmpty());
```

Comment:
Ditto for account number and PIN. You could write an `askUserForInteger`, or just treat both of them as strings.